pkg/autoscaler/autoscaler.go

```diff
@@ -125,8 +127,9 @@ type Autoscaler struct {
 }
 
 // New creates a new instance of autoscaler
-func New(dynamicConfig *DynamicConfig, containerConcurrency v1alpha1.RevisionContainerConcurrencyType, reporter StatsReporter) *Autoscaler {
+func New(config *Config, dynamicConfig *DynamicConfig, containerConcurrency v1alpha1.RevisionContainerConcurrencyType, reporter StatsReporter) *Autoscaler {
```

Comment:
Why do we need to thread this through? All the other static values are read using the `Current()` method in line 171 or 174 respectively as well. What's different now?